### PR TITLE
Added a convenience function for plotting 2D points.

### PR DESCRIPTION
**Pull request by magicbycalvin**
Included a convenience function for plotting 2D points in Python. The new code closely resembles the existing plotting functions available in `gtsam.utils.plot`. Plotting a 2D point is useful for visualizing the location of landmarks.

### CODE REVIEW

File: python/gtsam/utils/plot.py
Comment (varunagrawal) [inline python/gtsam/utils/plot.py:128]: Kill this line? And maybe add a comment on what `k` is?

**Review by varunagrawal — APPROVED** [review]
LGTM but please address my comment before merging.

File: python/gtsam/utils/plot.py
Comment (magicbycalvin) [reply python/gtsam/utils/plot.py]: I removed the commented out line and included a comment as to what `k` is (the "sigma" value for the uncertainty ellipse). I specifically chose 5 sigma as that is the same value used in the other functions, such as `plot_pose2_on_axes`.
Thanks!

**Comment by dellaert** [comment]
@varunagrawal I'll let you merge

File: python/gtsam/utils/plot.py
Comment (senselessDev) [inline python/gtsam/utils/plot.py:128]: @magicbycalvin This is not correct in 2D, your values seem to be for the 1D case. See `Geometric interpretation` [here](https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Properties) and read [this](https://stackoverflow.com/a/39749274/1959116). Can you fix that in another PR?

Comment (dellaert) [reply python/gtsam/utils/plot.py:128]: Actually, I added a note in #1063, and I think k=5 indeed corresponds to 99.9996% in 2D case. Check the note and let me know... That PR could be amended to accommodate a change if needed, if @gchenfc does not merge it first.

Comment (dellaert) [reply python/gtsam/utils/plot.py:128]: It's in plot.py, in that PR

Comment (magicbycalvin) [reply python/gtsam/utils/plot.py:128]: Just now seeing this discussion. I have since added an additional PR addressing the comments above.

Comment (senselessDev) [reply python/gtsam/utils/plot.py:128]: > Actually, I added a note in #1063, and I think k=5 indeed corresponds to 99.9996% in 2D case. Check the note and let me know... That PR could be amended to accommodate a change if needed, if @gchenfc does not merge it first.

While I agree with 99.9996% (see #1067), that is not what was written in the comment:

> to a 99.9999% confidence, i.e. assuming the estimation has been computed properly, there is a 99.999% chance that the true position

**Pull request merged**
